### ISSUE TITLE
Recognition of combinations of numbers and uppercase letters

### ISSUE THREAD

**Issue by LittleWhite0208** [ViTAE-Transformer/DeepSolo#10]
Hello, I reproduced the DeepSolo algorithm on the Chinese dataset of the document. The overall accuracy rate is not bad, but the field recognition of the combination of numbers and uppercase letters is not good, and the problem of missing characters is prone to occur. The length of these texts is about 20. I tried to increase NUM_POINTS from 25 to 50, but there is still not much improvement. If it continues to increase, the speed will be slow. Have you encountered a similar situation? Thank you！

**Comment by ymy-k** [comment]
More information please~batchsize? loss setting? convergence? Posting some visualization imgs here will also help.

**Comment by LittleWhite0208** [comment]
The BATCHSIZE is set to 4, BASE_LR and LR_BACKBONE start to be set to 1e-4, and STEPS is set to (200k, 300k). A total of 400k steps have been trained, and the loss will no longer drop when it drops to about 2. 

Since the data involves privacy, it is inconvenient to display, so let me give an example. For example, the number to be recognized is 12345678111112345678, and the model is easy to recognize as 123456781112345678. I feel that it may be affected by the maximum decoding length of ctc. I increased NUM_POINTS from 25 to 50 and it improved, but it didn't solve it completely. Continuously increasing the number of sampling points will cause the model inference speed to be too slow. Is there any other trick to solve this problem?

**Comment by ymy-k** [comment]
Increasing the text loss weight after some training steps may help.
